features/performance/assets/pre-boot-tuning.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# TODO: improve check for applied configuration
+if grep -q "CPUAffinity" "/etc/systemd/system.conf"; then
+    echo "Pre boot tuning configuration already applied"
+    # Move kernel rcuo* threads to the housekeeping cpus
+else
+    echo "[Manager]" >> /etc/systemd/system.conf
+    echo "CPUAffinity=0 2" >> /etc/systemd/system.conf
+    echo "IRQBALANCE_BANNED_CPUS=FFFFFFFA" >> /etc/sysconfig/irqbalance
```

Comment:
These two needs to be calculated as well, based on the isolcpus and reservedSystemCPUs that we set in #23

Reply:
we can do that if we set in #23  the reservedCPU value as:
kernelArguments:
  - non_iso_cpu=$reservedCPU 
This means we need to ensure your script runs first